src/cve/data_models/cve_intel.py

```diff
@@ -24,20 +25,31 @@
 from pydantic import Field
 
 
+def rgetattr(obj, attr, *args):
+    def _getattr(obj, attr):
+        return getattr(obj, attr, *args)
+    return functools.reduce(_getattr, [obj]+attr.split('.'))
```

Comment:
Minor nit, spaces around operators
```suggestion
    return functools.reduce(_getattr, [obj] + attr.split('.'))
```